boards/vcc-gnd/yd_stm32h750vb/yd_stm32h750vb.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: yd_stm32h750vb
+name: YD-STM32H750VB
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 1024
+flash: 128
+supported:
+  - gpio
+  - dma
```

Comment:
This could be reviewed I guess

Reply:
Do we need more information to describe here？

Reply:
Idea is to mention what are the devices/nodes enabled on the board